public/app/features/scopes/selector/ScopesSelectorService.test.ts

```diff
@@ -0,0 +1,266 @@
+import { Scope } from '@grafana/data';
```

Comment:
There are quite a few tests for scopes in general that are more "integration" ie testing whole dashboard page but wanted to start adding some unit tests for the services too.